docs/src/content/core/color.mdx

```diff
@@ -3,7 +3,7 @@ name: Color
 route: /core/color
 ---
 
-import { ColorCategories } from '../../components/color/color-categories'
+import { Colors } from './color'
```

Comment:
I'm getting a warning on run yarn start:dev saying this import is not found. Color page is no longer rendering. I ran gatsby clean and the result is the same. 
<img width="905" alt="Screen Shot 2020-10-07 at 9 08 37 AM" src="https://user-images.githubusercontent.com/1058725/95357753-1ddeb180-087d-11eb-919b-d4205a32f5f4.png">

Reply:
does a gatsby clean help for you?  I get this error too and that helps me.

Reply:
eh, scratch that.  Just tried again.  I still get the error.  Will address...